### ISSUE TITLE
AOT causes problem with clojurescript

### ISSUE THREAD

**Issue by Frozenlock** [dscarpetti/codax#19]
When using 1.2.0-snapshot and running figwheel, I get this error : 

```
  No reader function for tag Inf

  986  (js-mod (Math/floor o) 2147483647)
  987  (case o
  988    ##Inf
              ^--- No reader function for tag Inf
  989    2146435072
  990    ##-Inf
  991    -1048576
```

However, re-adding `^:skip-aot` solve this.

**Comment by dscarpetti** [comment]
Are you using the latest version of figwheel? I found possible reference to this issue here: https://clojurians-log.clojureverse.org/lein-figwheel/2017-10-04.html

**Comment by Frozenlock** [comment]
Yes, I already tried to require `[org.clojure/tools.reader "1.1.0"]`, but I got the same error.

I narrowed down the source of the problem to requiring both codax (with AOT) and cljs-ajax. Any of those alone is fine, but combined they produce this error.

---
To reproduce from a new project : 


1. `lein new reagent my-project`

2. Add the following dependencies:
```clj
[codax "1.2.0"]
[cljs-ajax "0.7.3"]
```

3. Run figwheel.
